services/order.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+from db.models import Ticket, Order, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
```

Comment:
Consider using a `datetime` object for `date` instead of a string to ensure proper handling and validation of date and time.